ReadMe.md

```diff
@@ -0,0 +1,215 @@
+
+```
+https://github.com/The-Normal-One1/bookstore
+
+```
+
+-Copy the URL.
+
+-In Visual Studio Code “Terminal” change the current working directory to the location where you want the cloned directory.
+
+-Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+```
+$ git clone {repository URL}
+```
+
+# Install Dependencies(with following Command)
+
+- npm install
+
+# Build webpack <a name="setup"></a>
+
+- run command
+
+```
+npm run build
+```
+
+## Start server
+
+```
+npm start
+```
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need:
+
+- [x] Basic Knowledge of **HTML**, **CSS**, & **JavaScript**.
+- [x] Basic knowledge of **ES6** syntax.
+- [x] Basic knowledge of **Git**, **GitHub** &**Gitflow**.
+- [x] Basic knowledge of **Linters**.
+- [x] Basic knowledge of **Webpack**.
+- [x] Basic knowledge of **VS Code** or any other code-editors.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+    _cd PATH_
+
+    _git clone  https://github.com/The-Normal-One1/bookstore.git
+
+_cd Javascript-Group-Capstone_
+
+_npm install_
+
+_npm run build_
+
+_npm start_
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using:
+
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Authors
+Amk lass
+
+GitHub:[@amklass11](https://github.com/amklass11)
+LinkedIn:[@AmkLAss](https://www.linkedin.com/in/amk-lass-521565196/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement additional pages**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/The-Normal-One1/bookstore/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to [Microverse](https://www.microverse.org/) for giving this opportunity to learn ...
+- Code Reviewers & Coding Partners.
+- Hat tip to anyone whose code was used.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] I would advise you to link MIT to the license. I am requesting this because it is part of the general requirements of the Project. Moreover, the License will make your Readme looks more professional. 👍